pkg/storage/blockdev.go

```diff
@@ -0,0 +1,391 @@
+// Copyright 2017-2019 the u-root Authors. All rights reserved
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package storage
+
+import (
+	"bufio"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"path"
+	"path/filepath"
+	"strconv"
+	"strings"
+
+	"github.com/rekby/gpt"
```

Comment:
not the u-root gpt package?

Reply:
I think I wrote this before the u-root gpt package (and if not, I probably just didn't see it)